src/todotasks.js

```diff
@@ -14,5 +14,39 @@ const showPage = (taskList) => {
     clear.forEach((task, index) => task.addEventListener('click', () => Task.delTask(index, taskList)));
     return taskList;
   });
+
+  const completechecker = () => {
+    const checked1 = document.querySelectorAll('input[type=checkbox]');
+
+    checked1.forEach((e) => {
+      e.addEventListener('change', () => {
+        if (e.checked === true) {
+          taskList[e.id].completed = true;
+        } else {
+          taskList[e.id].completed = false;
+        }
+        localStorage.setItem('taskDetail', JSON.stringify(taskList));
+      });
+    });
+  };
+
+  const clearAll = document.querySelector('.clear-items');
+  clearAll.addEventListener('click', () => {
```

Comment:
- [ ] Kindly fix the `clear all completed` functionality because now it's not working correctly. It always removes the first task.
hint: You can fix it by using the `completed` property in the object for filter instead of `Number(c1)` because `Number(c1)` always return 0.